apps/server/src/config/appConfig.ts

```diff
@@ -66,6 +67,10 @@ const buildAppContext = async (secrets: any): Promise<AppConfig> => {
 			return process.env.OPENAPI_PATH || '/api-docs';
 		},
 
+		corsAllowedOrigins(): string[] {
+			return process.env.CORS_ALLOWED_ORIGINS ? process.env.CORS_ALLOWED_ORIGINS.split(',') : [];
```

Comment:
either the "no spaces" in readme, or this here?

```suggestion
			return process.env.CORS_ALLOWED_ORIGINS ? process.env.CORS_ALLOWED_ORIGINS.split(',').map(trim) : [];
```

Reply:
this is fine idea, I'll add it.